src/test/kotlin/org/ionproject/integration/application/job/tasklet/WriteFileTaskletTests.kt

```diff
@@ -111,20 +108,6 @@ class WriteFileTaskletTests {
         writeFileTaskletFailure = WriteFileTasklet(mockDispatcherFailure)
     }
 
-    @Test
-    fun `when write file to disk then if file exists returns success`() {
```

Comment:
This test doesn't make sense since we aren't writing to disk a file with the business objects serialized.